examples/pyproject.toml

```diff
@@ -18,6 +18,8 @@ dependencies = [
     # 1.4.1 adds version constraints to scikit-learn and numpy that are incompatible with ART
     "albumentations<1.4.1",
     "armory-library",
+    "adversarial-robustness-toolbox",
+    "tidecv",
 ]
 
 [project.optional-dependencies]
```

Comment:
We could make all these optional dependencies part of the standard dependencies for armory-examples. I think the original idea was that the collection of examples might have a demonstrations for a large number of integrated libraries, but someone (other than a TwoSix armory developer) interested in the examples might not want to install _everything_.

Reply:
We might consider grouping armory-examples by application type (CV, NLP, etc) to partition dependencies.